Sequencer.cs

```diff
@@ -16,39 +16,99 @@ namespace WordSequence
 {
     public class Sequencer : CustomPwGenerator
     {
-        private string GetConfigurationPath()
+        /* if returnEmptyConfig is true, or a user config file is found, returns
+         * the path to the user config file.
+         *
+         * if returnEmptyConfig is false, or the user config file path is not
+         * specified, and the user config file is not found, returns the path to
+         * the global config.

```

Comment:
I clarified the comment and variable name. This function is the mechanism for creating a user config by copying the default config. It accomplishes this transparently by returning the default config path for reading, but the user path for writing, when the user path points to a non-existing file.